staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -596,7 +602,8 @@ func (meal *cfgMeal) presyncFlowSchemaStatus(fs *fctypesv1a1.FlowSchema, isDangl
 // imaginePL adds a priority level based on one of the mandatory ones
 func (meal *cfgMeal) imaginePL(proto *fctypesv1a1.PriorityLevelConfiguration, requestWaitLimit time.Duration) {
```

Comment:
why `imaginePL`? why not `addPriorityLevel`?

Reply:
Expanded the comment on that function, hopefully making the answer clear.